GVFS/GVFS.FunctionalTests/Tests/FastFetchTests.cs

```diff
@@ -141,11 +141,8 @@ public void ForceCheckoutRequiresCheckout()
         {
             this.RunFastFetch($"--checkout --folders \"/Scripts\" -b {Settings.Default.Commitish}");
 
-            // Run a second time in the same repo on the same branch with more folders. 
-            string result = this.RunFastFetch($"--force-checkout --folders \"/GVFS;/Scripts\" -b {Settings.Default.Commitish}");
-
-            string[] expectedResults = new string[] { "Cannot use --force-checkout option without --checkout option." };
-            result.ShouldContain(expectedResults);
+            // Run a second time in the same repo on the same branch with more folders but expect an error.

```

Comment:
@kyle-rader, could you update this comment to explain why we're expecting an error?

(I'm not sure why the check you removed was failing)